### PR TITLE
Added toggle to decide whether extensions should be used

### PR DESCRIPTION
**Pull request by AndrewBastin**
This PR intends to add a toggle in the Settings page to turn off and on the use of extensions to send requests.

**NOTE for Translators:**
New entries added in the translations file

### CODE REVIEW

**Comment by liyasthomas** [comment]
This is handy ✌️

File: functions/network.js [hunk 1]
```diff
@@ -2,16 +2,25 @@ import AxiosStrategy from "./strategies/AxiosStrategy";
 import FirefoxStrategy from "./strategies/FirefoxStrategy";
 import ChromeStrategy, { hasChromeExtensionInstalled } from "./strategies/ChromeStrategy";
 
+const isExtensionsAllowed = ({ state }) => {
+  console.log(typeof(state.postwoman.settings.EXTENSIONS_ENABLED) === 'undefined'
```
Comment (liyasthomas) [inline functions/network.js]: Please comment out / remove `console.log()`

Comment (AndrewBastin) [reply functions/network.js]: Oops, forgot about that 😛

Comment (AndrewBastin) [reply functions/network.js]: Okay, fixed that in the latest commit.

**Comment by AndrewBastin** [comment]
@liyasthomas bump.

**Comment by liyasthomas** [comment]
@AndrewBastin merge this when you're ready.

**Pull request merged**